Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -1431,6 +1431,32 @@ void NetworkConnectionToWebProcess::installMockContentFilter(WebCore::MockConten
 }
 #endif
 
+void NetworkConnectionToWebProcess::logOnBehalfOfWebContent(const String& logString, bool error)
+{
+    CString string = logString.ascii();
+
+    constexpr unsigned maxLength = 128;
+    auto stringLength = string.length();
+
+    if (stringLength > maxLength) {
+        string.mutableData()[maxLength] = 0;
+        stringLength = maxLength;
+    }
+
+    for (unsigned i = 0; i < stringLength; i++) {
+        char c = logString[i];
+        if (c >= 32 && c <= 126)
```

Comment:
This character validation is used in a few places in WebKit code. Might be nice to make a helper function someday.

Reply:
I will see if I can identify the other places this is done, and look into creating a helper function.